dashboard/app/models/levels/aichat.rb

```diff
@@ -0,0 +1,76 @@
+# == Schema Information
+#
+# Table name: levels
+#
+#  id                    :integer          not null, primary key
+#  game_id               :integer
+#  name                  :string(255)      not null
+#  created_at            :datetime
+#  updated_at            :datetime
+#  level_num             :string(255)
+#  ideal_level_source_id :bigint           unsigned
+#  user_id               :integer
+#  properties            :text(4294967295)
+#  type                  :string(255)
+#  md5                   :string(255)
+#  published             :boolean          default(FALSE), not null
+#  notes                 :text(65535)
+#  audit_log             :text(65535)
+#
+# Indexes
+#
+#  index_levels_on_game_id    (game_id)
+#  index_levels_on_level_num  (level_num)
+#  index_levels_on_name       (name)
+#
+
+# TODO: Update this level to account for the params we actually need.
```

Comment:
Callout: I did not do much work to customize this, but I assume that's all right to be future work.

Reply:
We don't want to lock ourselves into a schema that requires a migration to address, but it's probably the contents of `serialized_attrs` that we might want to adjust, and those can be changed without a migration since they all serialize to the same database field.